test/vm.cpp

```diff
@@ -365,13 +365,13 @@ void doVMTests(json_spirit::mValue& v, bool _fillin)
 		auto argc = boost::unit_test::framework::master_test_suite().argc;
 		auto argv = boost::unit_test::framework::master_test_suite().argv;
 		for (auto i = 0; i < argc; ++i)
-		{	       
+		{
 			if (std::string(argv[i]) == "--show-times")
 			{
 				auto testDuration = endTime - startTime;
 				cnote << "Execution time: "
-				      << std::chrono::duration_cast<std::chrono::milliseconds>(testDuration).count()
-				      << " ms";
+					  << std::chrono::duration_cast<std::chrono::milliseconds>(testDuration).count()
```

Comment:
still some spaces here (note follow-on lines need not line up - one tab is enough).